tools/rdm/TestMixins.py

```diff
@@ -59,16 +60,32 @@ def Test(self):
 class GetMixin(object):
   """GET Mixin for an optional PID. Verify EXPECTED_FIELD is in the response.
 
-    This mixin also sets a property if PROVIDES is defined.  The target class
-    needs to defined EXPECTED_FIELD and optionally PROVIDES.
+    This mixin also sets one or more properties if PROVIDES is defined.  The
+    target class needs to defined EXPECTED_FIELD and optionally PROVIDES.
+
+    If ALLOWED_NACKS is non-empty, this adds a custom NackGetResult to the list of
+    allowed results for each entry.
   """
+  ALLOWED_NACKS = []
+
   def Test(self):
-    self.AddIfGetSupported(self.AckGetResult(field_names=[self.EXPECTED_FIELD]))
+    if isinstance(self.EXPECTED_FIELD, list):
+      expected_fields = self.EXPECTED_FIELD
```

Comment:
Same comment as before here, we should make this EXPECTED_FIELDS and avoid the use of isinstance . Do this in another change though - this one is already pretty big